tests/openvino/test_quantization.py

```diff
@@ -451,30 +482,30 @@ def test_ov_model_static_quantization_with_auto_dataset(
             ov_model = model_cls.from_pretrained(model_id, quantization_config=quantization_config)
             ov_model.save_pretrained(tmp_dir)
 
+            submodels = ov_model.ov_submodels.values()
+
             if model_cls == OVModelForSpeechSeq2Seq:
-                submodels = [ov_model.encoder.model, ov_model.decoder.model]
-                if ov_model.decoder_with_past is not None:
-                    submodels.append(ov_model.decoder_with_past.model)
-                else:
+                if ov_model.decoder_with_past is None:
                     expected_num_weight_nodes_per_model = expected_num_weight_nodes_per_model[:-1]
                     expected_fake_nodes_per_model = expected_fake_nodes_per_model[:-1]
 
                 input_features = torch.randn((1, 128, 3000), dtype=torch.float32)
                 ov_model.generate(input_features)
-            elif model_cls == OVModelForCausalLM:
-                submodels = [ov_model]
-
+            elif model_cls in (OVModelForCausalLM, OVModelForFeatureExtraction):
                 tokenizer = AutoTokenizer.from_pretrained(model_id)
                 if tokenizer.pad_token is None:
                     tokenizer.pad_token = tokenizer.eos_token
                 tokens = tokenizer("This is a sample input", return_tensors="pt")
-                outputs = ov_model(**tokens)
-                self.assertTrue("logits" in outputs)
```

Comment:
why is this removed ?

Reply:
at least it runs the quantized model.. isn't the idea behind this test is to load with quantization and run inference sanity checks ?

Reply:
okay I see the `ov_model(**tokens)` afterwards

Reply:
The model is still run, the only thing missing is `self.assertTrue("logits" in outputs)`